src/vanilla/Model/ParameterTS.cs

```diff
@@ -47,5 +47,22 @@ public IEnumerable<Core.Model.Property> ComposedProperties
 
         [JsonIgnore]
         public bool IsLocal => (!IsClientProperty) && !string.IsNullOrWhiteSpace(Name) && !IsConstant;
+
+        [JsonIgnore]
+        public string OperationType
+        {
+            get
+            {
+                string operationType = null;
+
+                IModelType modelType = ModelType;
```

Comment:
minor thing, but I'd remove this local and the blank line on 57

Reply:
Done in https://github.com/Azure/autorest.typescript/pull/95/commits/8a706f5e415a9d40e12642b1d3276f6f632cd831.